src/yara.cc

```diff
@@ -467,8 +454,6 @@ class AsyncConfigure : public Nan::AsyncWorker {
 		} catch(std::exception& error) {
 			SetErrorMessage(error.what());
 		}
```

Comment:
Do you really want to catch all exceptions here?

Reply:
Those catch all were in the original code, we should probably improve them but I'll leave it for another PR so this one doesn't get too big.